float32 std_beta
float32 high_beta 
float32 low_beta